ui/src/criteria/concept.tsx

```diff
@@ -92,7 +92,7 @@ class _ implements CriteriaPlugin<Data> {
     return <ConceptDetails data={this.data} />;
   }
 
-  generateFilter(entityVar: string) {
+  generateFilter(entityVar: string, fromOccurrence: boolean) {
```

Comment:
nit: After reading the code I'm still a bit unclear on the semantics of `fromOccurence`, consider adding a comment here